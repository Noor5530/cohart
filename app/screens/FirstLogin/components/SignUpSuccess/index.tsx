import React from "react";

import { View, Text, Image } from "react-native";
import ArrowDown from "assets/Arrow_Down.png";

import styles from "./style";

const SignUpSuccess: React.FC = () => {
  return (
    <View style={styles.container}>
      <Text style={styles.headingText}>You are all set!</Text>
      <Text style={styles.headingText}>
        Keep discovering your new art world
      </Text>
      <Text style={styles.subText}>Let&apos;s go</Text>
      <Image style={styles.icon} source={ArrowDown} />
    </View>
  );
};

export default SignUpSuccess