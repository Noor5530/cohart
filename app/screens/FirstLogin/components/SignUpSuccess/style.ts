import { appFonts } from "components/text";
import { StyleSheet } from "react-native";
import {
  heightPercentageToDP as hp,
  widthPercentageToDP as wp,
} from "react-native-responsive-screen";

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: "white",
    paddingTop: hp("25"),
    paddingRight: wp("10"),
    paddingLeft: wp("10"),
  },
  headingText: {
    fontFamily: appFonts.InterBold,
    fontSize: hp("6"),
  },
  subText: {
    textTransform: "uppercase",
    fontFamily: appFonts.InterSemiBold,
    fontSize: hp("3"),
    alignSelf: "center",
    marginTop: hp("10"),
    marginBottom: hp("2.7"),
  },
  icon: {
    alignSelf: "center",
  },
});

export default styles;
