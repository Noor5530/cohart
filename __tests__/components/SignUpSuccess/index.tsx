import "react-native";
import React from "react";
import renderer from "react-test-renderer";

import SignUpSuccess from "../../../app/screens/FirstLogin/components/SignUpSuccess";
jest.mock("react-native-vector-icons/Entypo", () => "Entypo");
jest.useFakeTimers();

test("renders correctly", () => {
  const tree = renderer.create(<SignUpSuccess />).toJSON();

  expect(tree).toMatchSnapshot();
});
