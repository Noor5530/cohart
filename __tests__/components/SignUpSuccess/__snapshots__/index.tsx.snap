// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "flex": 1,
      "paddingLeft": undefined,
      "paddingRight": undefined,
      "paddingTop": undefined,
    }
  }
>
  <Text
    style={
      Object {
        "fontFamily": "Inter-Bold",
        "fontSize": undefined,
      }
    }
  >
    You are all set!
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "Inter-Bold",
        "fontSize": undefined,
      }
    }
  >
    Keep discovering your new art world
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "center",
        "fontFamily": "Inter-SemiBold",
        "fontSize": undefined,
        "marginBottom": undefined,
        "marginTop": undefined,
        "textTransform": "uppercase",
      }
    }
  >
    Let's go
  </Text>
  <Image
    source={
      Object {
        "testUri": "../../../app/assets/Arrow_Down.png",
      }
    }
    style={
      Object {
        "alignSelf": "center",
      }
    }
  />
</View>
`;
